### ISSUE TITLE
Opening deep link removes chat tab item from app menu

### ISSUE THREAD

**Issue by phpmaps** [microsoft/botbuilder-js#2894]
## Versions
**Package version:**  botbuilder ^4.10.4,
**Nodejs version:** v12.3.1
**Browser version:** _NA_ (Teams Desktop)
**What os:** Happens in Win and OSX

## Describe the bug
Opening a deep link from a card into a tab extension causes the Chat tab to disappear from the menu.

**_Chat is there_**

![image](https://user-images.githubusercontent.com/2159502/95949114-ddb08f00-0da6-11eb-9941-195caa61772c.png)

**_Chat is not there_**. _Where did it go?_

![image](https://user-images.githubusercontent.com/2159502/95949563-c7570300-0da7-11eb-8aff-169fe46b4538.png)

Why does chat tab menu item disappear after deep link opens tab? 

## To Reproduce
1. Call _createViewerStaticDeepLink()_ making sure APPID, TAB_ENTITY_ID are set 
2. Use the URI string to openUrl button action, ie:

```
const hero = CardFactory.heroCard(
                "Dev Question",
                "Where does chat tab go?",
                null,
                CardFactory.actions([
                    {
                        type: "openUrl",
                        title: "Open in Viewer",
                        value: createViewerStaticDeepLink()
                        )
                    }
                ])
            );
```

3. Observe that Chat menu item will disappear as soon as the deep link opens the tab.

## Expected behavior
Deep linking should keep tab positions as they are and not remove tab items, ie Chat, File, My Tab, Viewer... from an app.  

## Code to generate deep link to static tab
```ts

// createViewerStaticDeepLink will call createDeepLink to generate URI

const APPID = <appId>;
const TAB_ENTITY_ID = <tabId>;

export function createViewerStaticDeepLink(subEntityId?: string) {
    return createDeepLink(APPID, TAB_ENTITY_ID, `{"subEntityId":"abc"}`);
}

function createDeepLink(appId: string, entityId: string, context: string) {
    return `https://teams.microsoft.com/l/entity/${appId}/${entityId}?context=${encodeURI(context)}`;
}


```

## Additional context
This behavior occurs in a personal app that has a bot extension and tab extension.

**Comment by Wajeed-msft** [comment]
Thanks for reporting this. This is a [known bug](https://domoreexp.visualstudio.com/MSTeams/_workitems/edit/1216459) and it's already fixed. The fix is rolled out in internal build and might take one or two weeks time to reach public rings.

**Comment by phpmaps** [comment]
@Wajeed-msft @anusharavindrar thanks.  will look for the update.

**Comment by phpmaps** [comment]
@Wajeed-msft -  Would the fix be included in 4.10.5 or 4.10.4?
